.github/workflows/production.yml

```diff
@@ -150,6 +150,7 @@ jobs:
       contents: write
     outputs:
       deploy_dev: ${{ steps.fast_forward_merge.outputs.deploy_dev }}
+      deploy_auth_dev: ${{ steps.fast_forward_merge.outputs.deploy_auth_dev }}
```

Comment:
This means whenever the production workflow runs i.e. when we are deploying / merged to main, then we fast forward those commits to the `env/dev/auth-dev` git branch.